web/rest/platform/config/api/raw/provider.yml

```diff
@@ -355,15 +355,19 @@ Ivoz\Provider\Domain\Model\ProxyTrunksRelBrand\ProxyTrunksRelBrand:
     access_control: '"ROLE_SUPER_ADMIN" in roles && user.hasAccessPrivileges(_api_resource_class, request.getMethod())'
 
 Ivoz\Provider\Domain\Model\ProxyUser\ProxyUser:
-  itemOperations:
-    get: ~
-    put: ~
-  collectionOperations:
-    get: ~
+  itemOperations: ~
+  collectionOperations: ~
   attributes:
     access_control: '"ROLE_SUPER_ADMIN" in roles && user.hasAccessPrivileges(_api_resource_class, request.getMethod())'
     order:
       name: ASC
+    write_access_control:
+      ROLE_SUPER_ADMIN:
+        raw: 'this.hasChanged("name") == false || this.getId() != 1'
```

Comment:
Not sure why this condition exists, it should have the same access controls as ProxyTrunks entity above :thinking: Maybe this condition should also be present in the ProxyTrunks endpoint?

Reply:
Those are the rules: default proxy user's name cannot be changed. It wasn't possible before either (but only at UI level).

I may agree with the second part but that would be part of another issue